oc_cdtapi/Dbsm2API.py

```
@@ -221,7 +219,7 @@ def login(self):
             access_token = resp_data['access_token']
         else:
             logging.error('Server returned an error [%s] [%s]' % (resp.status_code, resp.text))
-            raise API.HttpAPIError(code=resp.status_code, text='login failed')
+            raise API.HttpAPIError('login failed')
```

Ruff (0.15.9)

[warning] Avoid specifying long messages outside the exception class

(TRY003)